### DISCUSSION TITLE
Sticky or frozen table columns

### DISCUSSION THREAD

**Discussion by jacobcoro** [filamentphp/filament#12036] [Ideas]
Freeze a column such that it is always visible on horizontal scroll.

**Comment by jarallahassalami** [comment]
Yes, it is very important

**Comment by zeeshantariq08** [comment]
I’m planning to implement sticky columns for tables. Would the core team accept a PR for this feature?

**Reply by mstfkhazaal** [reply]
any update?

**Reply by zeeshantariq08** [reply]
https://github.com/zeeshantariq08/filament-sticky-columns
